docs/snippets/proc-deploying-function.md

```diff
@@ -0,0 +1,58 @@
+<!-- Snippet used in the following topics:
+- /docs/getting-started/build-run-deploy-func.md
+- /docs/functions/build-run-deploy-func.md
+-->
+The `deploy` command uses the function project name and the image registry name to construct a fully qualified image name for your function.
```

Comment:
```suggestion
The `deploy` command uses the function project name as the Knative Service name. When your function is built, the project name and the image registry name are used to construct a fully qualified image name for your function.
```

Some minor clarifications. The actual image name is derived during the build step, not deploy.